### PR TITLE
Put HELM instructions into H2O download page

### PR DESCRIPTION
**Pull request by Pscheidl**

### CODE REVIEW

File: h2o-dist/index.html [hunk 1]
```diff
@@ -677,7 +677,19 @@ <h2>Gradle-style specification for Maven artifacts</h2>
 }</br>
           </div>
             <div id="quickstart-kubernetes" style="display:none">
-                <h2>Get started with H<sub>2</sub>O on Kubernetes</h2>
+
+                <h2>Setup H<sub>2</sub>O on Kubernetes using Helm</h2>
+<a href="https://helm.sh/" target="_blank">Helm</a> can be used to deploy H2O into a kubernetes cluster. Helm requires the KUBECONFIG environment variable to be set up properly, or stating the kubeconfig destination explicitly. Please refer to Helm's documentation
```
Comment (honzasterba) [inline h2o-dist/index.html:682]: should text be inside a <p>, also would prefet to fix the indent and line length

File: h2o-dist/index.html [hunk 2]
```diff
@@ -677,7 +677,19 @@ <h2>Gradle-style specification for Maven artifacts</h2>
...
+    for further information.
+                <p class="terminal" id="to_copy11">
+helm repo add h2o https://charts.h2o.ai --version SUBST_PROJECT_VERSION
+helm install basic-h2o h2o/h2o
+helm test basic-h2o
+                </p>
+
+There are various settings and modifications available. To inspect the configuration options available, use the  "helm inspect values h2o/h2o --version SUBST_PROJECT_VERSION" command.
```
Comment (honzasterba) [inline h2o-dist/index.html:690]: should text be inside a <p>, also would prefet to fix the indent and line length

**Comment by honzasterba** [comment]
should this go to release branch?

File: h2o-dist/index.html (hunk 1 above)
Comment (Pscheidl) [reply h2o-dist/index.html]: Put all of into a paragraph and that fixed the indetation :wink: Thanks.

File: h2o-dist/index.html (hunk 2 above)
Comment (Pscheidl) [reply h2o-dist/index.html]: Put all of into a paragraph and that fixed the indetation wink Thanks.

File: h2o-dist/index.html [hunk 3]
```diff
@@ -677,7 +677,19 @@ <h2>Gradle-style specification for Maven artifacts</h2>
 }</br>
           </div>
             <div id="quickstart-kubernetes" style="display:none">
-                <h2>Get started with H<sub>2</sub>O on Kubernetes</h2>
+
+                <h2>Setup H<sub>2</sub>O on Kubernetes using Helm</h2>
+                    <p></p><a href="https://helm.sh/" target="_blank">Helm</a> can be used to deploy H2O into a kubernetes cluster. Helm requires the KUBECONFIG environment variable to be set up properly, or stating the kubeconfig destination explicitly. Please refer to Helm's documentation
```
Comment (honzasterba) [inline h2o-dist/index.html:682]: the closing `</p>` seems like a mistake here

Comment (Pscheidl) [reply h2o-dist/index.html]: :man_facepalming:

**Review by michalkurka — APPROVED** [review]

**Pull request merged**